tests/bson/bson-utcdatetime-001.phpt

```diff
@@ -12,21 +12,21 @@ $manager = new MongoDB\Driver\Manager(STANDALONE);
 
 $classname = BSON_NAMESPACE . "\\UTCDateTime";
 $utcdatetime = new $classname("1416445411987");
-$result = $manager->executeInsert(NS, array('_id' => 1, 'x' => $utcdatetime));
-$query = new MongoDB\Driver\Query(array('_id' => 1));
-$cursor = $manager->executeQuery(NS, $query);
-$array = iterator_to_array($cursor);
```

Comment:
Please don't change test cases when changing code that affects the test. That makes it really difficult to see whether you broke the test case, or the code.